include/oboe/StreamBuilder.h

```diff
@@ -199,7 +202,7 @@ class OboeStreamBuilder : public OboeStreamBase {
      * @param streamCallback
      * @return
      */
-    OboeStreamBuilder *setCallback(OboeStreamCallback *streamCallback) {
+    StreamBuilder *setCallback(StreamCallback *streamCallback) {
```

Comment:
Please consider specifying StreamCallback ownership via use of std::unique_ptr or std::shared_ptr. With a raw pointer, it's easy to run into a trouble of the client code deletes the callback instance.

Reply:
Good spot. I think it makes sense for the client to retain ownership of the callback so that means using a `std::shared_ptr<StreamCallback>`. I've updated the code to do this. Would you mind taking a look.